src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,28 +4,38 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
-			return null;
-		}
-		StringBuilder builder = new StringBuilder(new String(make(n, n)));
-		for (int i = n - 2; i > 0; i -= 2) {
-			char[] chars = make(n, i);
+	private final char WHITESPACE = ' ';
+
+	private final char STAR = '*';
+
+	private final Character NEW_LINE = '\n';
+
+	public String print(int width) {
```

Comment:
![bg,f8f8f8-flat,750x,075,f-pad,75x100,f8f8f8 u7](https://github.com/modernSE/kata-diamond-java/assets/28872997/e0e76a17-d530-4ecf-a8b1-e2d5c59759b1)

this doesnt print but just builds the string